src/framework/annotator_mgr.cpp

```diff
@@ -379,10 +391,10 @@ namespace uima {
 
 
     bool AnnotatorManager::shouldEngineBeCalled(uima::internal::CapabilityContainer const & crCapContainer,
-        ResultSpecification const & rResultSpec,
-        Language const & crLanguage,
-        vector<TypeOrFeature> & rTOFSToBeRemoved) {
-      util::Trace                 clTrace(util::enTraceDetailHigh, UIMA_TRACE_ORIGIN, UIMA_TRACE_COMPID_ANNOTATOR_MGR);
+                                                ResultSpecification const &rResultSpec,
```

Comment:
I don't see you have changed anything here aside from whitespace. Revert unnecessary changes.